sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -24,10 +24,10 @@
 import re
 import time
 
+from StringIO import StringIO
 
 from apitools.base.py import encoding
 from apitools.base.py import exceptions
-
```

Comment:
Dividing the imports into blocks is probably intentional. Imports must be ordered within a block.